wandb/sdk/wandb_settings.py

```diff
@@ -94,13 +95,14 @@
     username=None,
     disable_code=None,
     anonymous=None,
+    ignore_globs="WANDB_IGNORE_GLOBS",
```

Comment:
ignore_globs=None (if it is a direct translation from key name)

Reply:
I'm not sure what to make of this comment?  Should we change something?

Reply:
ignore_globs=None,
will make it so WANDB_IGNORE_GLOBS gets passed through to this setting.

ignore_globs="WANDB_IGNORE_GLOBS" is redundant.   assigning to a string is only used if you are renaming environment variable to setting variable.